src/ray/core_worker/lib/java/io_ray_runtime_gcs_GlobalStateAccessor.cc

```diff
@@ -116,6 +116,23 @@ Java_io_ray_runtime_gcs_GlobalStateAccessor_nativeGetActorInfo(JNIEnv *env, jobj
   return nullptr;
 }
 
+JNIEXPORT jbyteArray JNICALL
+Java_io_ray_runtime_gcs_GlobalStateAccessor_nativeGetActorAddress(JNIEnv *env, jobject o,
+                                                                  jlong gcs_accessor_ptr,
+                                                                  jbyteArray actorId) {
+  const auto actor_id = JavaByteArrayToId<ActorID>(env, actorId);
+  auto *gcs_accessor =
+      reinterpret_cast<ray::gcs::GlobalStateAccessor *>(gcs_accessor_ptr);
+  auto actor_info = gcs_accessor->GetActorInfo(actor_id);
```

Comment:
I believe getting from GCS every invocation is too heavy. But it works as python does. And I have no approach either now.

Reply:
Can we cache this information? Does actor's address change?

Reply:
Actor address would be changed once it failed over. If we cache them, we should subscribe the change to update local cache.

BTW, I don't think it's necessary to addrss this issue in this PR, because this is aligned to Python implementation.